packages/docs/pages/tabs.tsx

```diff
@@ -38,12 +38,18 @@ const TabsPage = () => {
 
             return (
               <>
-                <Tabs activeTab={activeTab} items={items} onTabClick={setActiveTab} />
+                <Tabs
+                  aria-label="Example Tab Content"
+                  activeTab={activeTab}
+                  items={items}
+                  id="tab-example"
+                  onTabClick={setActiveTab}
+                />
                 <Box marginTop="large">
-                  {activeTab === 'tab1' && <Text>Content 1</Text>}
-                  {activeTab === 'tab2' && <Text>Content 2</Text>}
-                  {activeTab === 'tab3' && <Text>Content 3</Text>}
-                  {activeTab === 'tab4' && <Text>Content 4</Text>}
+                  {activeTab === 'tab1' && <Text id="tab1-content">Content 1</Text>}
+                  {activeTab === 'tab2' && <Text id="tab2-content">Content 2</Text>}
+                  {activeTab === 'tab3' && <Text id="tab3-content">Content 3</Text>}
+                  {activeTab === 'tab4' && <Text id="tab4-content">Content 4</Text>}
                 </Box>
```

Comment:
@MariaJose brings out a good point. 

Can we also add a warning if the `id` is not being used? I don't know how feasible this is to implement. I also don't know if the browser does this by default.

Reply:
Yeah, probably adding a more detailed note in this example (see example above). And a general one in the `do`s section? thoughts?

<img width="968" alt="Screen Shot 2022-03-16 at 1 42 00 PM" src="https://user-images.githubusercontent.com/39739180/158663904-0abf98e7-0a5a-46e1-8c6d-4ab8c24d2237.png">

Reply:
@rtalvarez Just to be clear, what I meant was if there is a way to notify developer (via a console warning, maybe), that the element with the `id` of `tab1-content` isn't in the dom? I don't know if the browser does this automatically.

Reply:
Do we do this anywhere else? It would warn on every re-render, I'm a bit hesitant to spam the console :|

Reply:
Good point. Honestly I don't think so? But I've seen libraries do it. I'm only concerned because I imagine some will not use ids haha.

Reply:
So we do have a `warning` method, like Reacts `warning`/`error` methods that only fire in development environments (so prod will be safe). We use it in our `FormGroup` component if you want to look at an example.
---
Ideally we'd add these accessibility controls for the user instead. Maybe it might be time to revisit this API as a `Tab.Item` approach? 🤔  Or potentially we ship a provider? Many options, but may be worth changing before we move out of "beta"?

Reply:
Problem with this API that I can think of, it is still up to the developer to pass in the "appropriate" component to the render prop that:

1. Accepts an `id` prop
2. Actually attaches that `id` to the DOM element

Reply:
Haven't messed around too much with API changes, but I was playing around with a warning for this. It feels kind of weird because I need to execute additional checks and DOM querying just to print a message, eg:

```javascript
    const showWarning = items.some((item) =>
      Boolean(!item.ariaControls && document.getElementById(`${item.id}-content`)),
    );

    if (showWarning) {
      warning(
        'TabItems must have an ariaControls field, otherwise, an element with fallback ID "{tab.id}-content" must exist in the DOM',
      );
    }
```

Is the overhead worth it? 🤔

Reply:
I think we should include it for now. At least there will be a warning for developers. I'd even consider making it an `error` to enforce it better 🤷